docs/data-sources/cdn_domains.md

```diff
@@ -0,0 +1,116 @@
+---
+subcategory: Content Delivery Network (CDN)
+---
+
+# huaweicloud_cdn_domains
+
+Use this data source to get a list of CDN domains.
+
+## Example Usage
+
+```hcl
+data "huaweicloud_cdn_domains" "test" {
+  type          = "web"
+  domain_status = "online"
+  service_area  = "mainland_china"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `domain_id` - (Optional, String) Specifies the ID of accelerate domain.
+
+* `name` - (Optional, String) Specifies the name of accelerate domain, using fuzzy matching.
+  The valid length is limited from **1** to **255**.
+
+* `type` - (Optional, String) Specifies the business type of accelerate domain.
+  The valid value can be **web**, **download**, **video** or **wholeSite**.
+
+* `domain_status` - (Optional, String) Specifies the status of accelerate domain.
+  The valid value can be **online**, **offline**, **configuring**, **configuring_failed**, **checking**,
+  **check_failed** or **deleting**.
+
+* `service_area` - (Optional, String) Specifies the area covered by the accelerate service.
+  The valid value can be **mainland_china**, **outside_mainland_china** or **global**.
+
+* `enterprise_project_id` - (Optional, String) Specifies the enterprise project ID of the CDN domain.
```

Comment:
The documentation mentions `"all" means all projects`.
If the document is correct, please add relevant description.